lib/worker.js

```diff
@@ -42,9 +42,15 @@ Communication.on('JOB', function(job) {
     configFile = params.configFile
   }
 
-  const modulesPath = find(params.fileDir, 'node_modules')
+  const packageJsonPath = find(params.fileDir, 'package.json')
   const eslintignoreDir = Path.dirname(find(params.fileDir, '.eslintignore'))
   let eslintNewPath = null
+  let modulesPath = null
+
+  if (packageJsonPath !== null) {
+    modulesPath = Path.join(Path.dirname(packageJsonPath), 'node_modules')
```

Comment:
What if there is no such folder, does this cause issues later on when it attempts to use it?